README.md

```diff
@@ -34,6 +34,17 @@ If for any reason some step in the command chain breaks, simply review the [Make
 
 ## Gotchas
 
+### Feature Flags
+
+When logging in as a creator, you may need to enable feature flags on that creator's account to be able to access the full set of UI options on the site.  Feature flags are stored as a json object on the Publisher model.  To update the flags for a user, run something like:
+```
+$ make docker-shell
+$ rails c
+$ p = Publisher.where(email: 'gemini@completed.org').first
+$ p.update!({ feature_flags: {"gemini_enabled"=>true, "bitflyer_enabled"=>true, "promo_lockout_time"=>"2020-11-23", "referral_kyc_required"=>true}})
```

Comment:
For working locally, these should be set for the users we test with. If they are not, let's make a follow-up PR to correct this. Thoughts?

Reply:
I'm not sure why it's disabled in local context, it seems to me like it's an oversight somewhere in the fixtures themselves because gemini is enabled by default in some method in the publishers model.

Reply:
I'll make a card and resolve this.

Reply:
Associated card: https://github.com/brave-intl/creators-private-issues/issues/73